scala/private/phases/phase_write_executable.bzl

```diff
@@ -85,12 +85,16 @@ def _write_executable_windows(ctx, executable, rjars, main_class, jvm_flags, wra
     cpfile = ctx.actions.declare_file("%s.classpath" % ctx.label.name)
     ctx.actions.write(cpfile, classpath)
 
+    specifiedEnv = {}
```

Comment:
Do we need the same change in both PRs? https://github.com/bazelbuild/rules_scala/pull/1502#discussion_r1242565719

Reply:
Both PRs need the change to work (I was trying to make the PRs standalone). But I could put in only one if thats what you want, probably not big deal if you are gonna commit both PRs soon. 
Please advise.

Reply:
@crt-31 we can merge PRs in order, #1502 and then #1503

Reply:
OK. I removed this 'specifiedEnv' change from this PR.
@liucijus  Ready for your review.